src/layouts/Lightwell.scss

```
@@ -0,0 +1,24 @@
+@use '~@patternfly/patternfly/sass-utilities/scss-variables' as *;
+
+// At xl+ (≥1200px), PF6 Page switches to a 2-column sidebar+main grid
+// and the masthead uses CSS subgrid. Lightwell has no sidebar, so force
+// single-column layout with a dedicated "subnav" row to keep the horizontal
+// navigation outside the page card.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)